formbuilder/components/FieldList.js

```diff
@@ -3,31 +3,53 @@ import { Link } from "react-router";
 import { Draggable } from "react-drag-and-drop";
 
 
+function MenuSection(props) {
+  const {heading, children} = props;
+  return (
+    <div className="panel panel-default">
+      <div className="panel-heading">{heading}</div>
+      <ul className="list-group">{children}</ul>
+    </div>
+  );
+}
+
 export default function FieldList(props) {
   return (
     <div>
-      <div className="list-group">
-        {
-          props.fieldList.map((field, index) => {
-            return (
-              <Draggable key={index} type="field"
+      <MenuSection heading="Widgets">{
+        props.fieldList.map((field, index) => {
+          return (
+            <Draggable key={index} type="field"
+              data={JSON.stringify(field)}
```

Comment:
Because we need to deserialize the string attached to the data attribute of the DOM element, and so retrieve the real schema definition object.